js/lightbeam.js

```diff
@@ -31,6 +31,9 @@ const lightbeam = {
     // initialize dynamic vars from storage
     if (!this.dataGatheredSince) {
       const { dateStr, fullDateTime } = await this.getDataGatheredSince();
+      if (!dateStr) {
```

Comment:
Seems pretty bad form to have such different shaped data, however this will fix the initial issue so will merge.